feconf.py

```diff
@@ -112,6 +112,7 @@
     TRAINING_JOB_STATUS_FAILED: [TRAINING_JOB_STATUS_NEW]
 }
 
+ENTITY_TYPE_STATE = 'exploration_state'
```

Comment:
Why is both this and exploration there? Also, is this needed? Entity_type would either be explorations or questions and not states, right?

Reply:
Oh my bad, I will remove this.

Reply:
Done